### PR TITLE
Dposse

### PR DESCRIPTION
**Pull request by dposse**
Added my about page, check it out?

### CODE REVIEW

File: client/assets/css/dan.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...n-top:3em}hr.post-end{width:40%}}@media(min-width:960px){.site-main{margin-top:6em}}@media(min-width:1300px){.site-main{margin-top:8em}.desktop-only,#toc.show-toc{display:block}.desktop-only-ib{display:inline-block}figure.left{margin-left:-240px}figure.left p{text-align:left}figure.right{margin-right:-240px}figure.right p{text-align:right}figure.big{width:1200px;margin-left:-240px}hr.post-end{width:30%}#toc{top:13em;margin-left:370px;max-width:220px}}@media(min-width:1800px){.site-main{margin-top:10em}.section-inner{max-width:1600px}.thin{max-width:960px}figure.left{max-width:75%;margin-left:-320px}figure.right{max-width:75%;margin-right:-320px}figure.big{width:1600px;margin-left:-320px}hr.post-end{width:30%}#toc{top:15em;margin-left:490px;max-width:300px}}@media(max-width:760px){.hide-in-mobile,.site-nav.hide-in-mobile{display:none}#menu-btn{display:inline-block}.posts-group{display:block}.posts-group .post-year{margin:-6px 0 4px}#spotlight.error-404{flex-direction:column;text-align:center}#spotlight.error-404 .banner-404{margin:0}}@media(max-width:520px){.content figure.left,.content figure.right{float:unset;max-width:100%;margin:0}hr.post-end{width:60%}#mobile-menu{right:1.2em}}
```
Comment (ravens-eye) [inline client/assets/css/dan.css:5]: It's best to load in normalize from a CDN, aggressive cache policy, better speed and all that.

Comment (ravens-eye) [reply client/assets/css/dan.css:5]: Several options here: https://classic.yarnpkg.com/en/package/normalize.css

File: client/dan.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          </svg></a>
+      </div>
+    </div>
+
+    <div>
+      <h3>The above is what my <a href="https://dposse.dev" target="_"><u>personal site</u></a> <em>should</em> look
+        like...
+        but there's
+        some
+        silly CORS
```
Comment (ravens-eye) [inline client/dan.html:53]: See this for fastest quick and dirty CORS workaround: https://github.com/izuzak/urlreq

See an example of how I've used it in the past: https://www.karstenrabe.dev/#github-scraper-code

**Review by ravens-eye — DISMISSED** [review]
Looks good! Just remove the npm lockfile cause we're using Yarn and should be good to go!

**Comment by ravens-eye** [comment]
I see now you're using Hugo... Not familiar with it and realize that normalize may be bundled with that ? Not sure. Would love to learn more

**Review by krab7191 — APPROVED** [review]

**Pull request merged**